diff --git a/node_modules/@quartz-community/crawl-links/dist/index.js b/node_modules/@quartz-community/crawl-links/dist/index.js
index 5005175..6ca9f7c 100644
--- a/node_modules/@quartz-community/crawl-links/dist/index.js
+++ b/node_modules/@quartz-community/crawl-links/dist/index.js
@@ -130,7 +130,10 @@ function transformLink(src, target, opts) {
         }
         const parts = slug2.split("/");
         const fileName = parts.at(-1);
-        return targetCanonical === fileName;
+        if (targetCanonical === fileName) {
+          return true;
+        }
+        return fileName === "index" && parts.at(-2) === targetCanonical;
       });
       if (matchingFileNames.length === 1) {
         const matchedSlug = matchingFileNames[0];
